src/airgap/README.md

```diff
@@ -0,0 +1,39 @@
+## Airgap bundle creation process
```

Comment:
Can we include instructions on folder from which to run these as it looks like they depend on specific folder structure?